prog/main.go

```diff
@@ -144,12 +144,37 @@ func (i *filterFlags) String() string {
 	}
 	return s
 }
+
 func (i *filterFlags) Set(value string) error {
 	*i = append(*i, value)
 
-	titleLabel := strings.Split(value, ":")
-	// TODO - allow escaping
-	v := app.CreateFilterOption(fmt.Sprintf("cmdlinefilter%d", len(*i)), titleLabel[0], render.HasLabel(titleLabel[1]))
+	//fmt.Printf("%q\n",value)
+	u := fmt.Sprintf("%q", value)
+	u = u[1 : len(u)-1]
+
+	soloColonFound := false
+	var title string
+	var label string
+	for k := 1; k < len(u); k++ {
```

Comment:
Why not simply use regexp `[^\\](:)` ? i.e. find a colon (first subgroup) not preceded by a backslash